Resources/doc/reference/configuration/proxy-client.rst

```diff
@@ -97,6 +93,20 @@ URL may contain a path. If you access your web application on a port other than
     Double-check ``base_url``, for if it is mistyped, no content will be
     invalidated.
 
+``tag_mode``
+"""""""""""""""""
+
+**type**: ``string`` **default**: ``ban``
+
+Select whether to invalidate tags using the xkey module or with ban requests. Supported modes: ``ban`` and ``purgekeys``.
+
+When using ``purgekeys`` the bundle will default to using soft purges.  If you do not want to use soft purge (either because your varnish modules version is too old to support it or because it does not fit your scenario), additionally set the tags_header option to xkey-purge instead of the default xkey-softpurge.
```

Comment:
imho this should link to the xkey varnish documentation. can we add a section like this:
 
```
To use the purgekeys method, you need the `xkey vmod`_ enabled and VCL to handle xkey invalidation requests as explained in the :ref:`FOSHttpCache library docs on xkey support <foshttpcache:varnish_tagging>`.